### PR TITLE
[CAS-997] Feature - Theme for AttachmentGalleryActivity

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-997](https://stream-io.atlassian.net/browse/CAS-997)

### 🎯 Goal

Add theme for AttachmentGalleryActivity.

### 🛠 Implementation details

Added a theme for each of the views inside `AttachmentGalleryActivity`.

### 🧪 Testing

Change the styles and check if the changes are applied.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes. (Done in #2046)
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Affected documentation updated (CMS, cookbooks, tutorial). (Done in #2046)
- [x] Reviewers added

### 🎉 GIF

giphy
_Fashion_

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -489,6 +487,50 @@
...
+
+    <style name="StreamUi.AttachmentGallery.CloseButton" parent="StreamUi">
+        <item name="android:src">@drawable/stream_ui_ic_close</item>
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginStart">@dimen/stream_ui_spacing_small</item>
+    </style>
+
+    <style name="StreamUi.AttachmentGallery.Title" parent="StreamUi">
+        <item name="android:textAppearance">@style/StreamUiTextAppearance.HeadlineBold</item>
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/res/values/styles.xml:508]: We are using our own `TextStyle` in other places - check `StreamUi.SuggestionListView` for example. Shouldn't we do the same in this Activity?

File: stream-chat-android-ui-components-sample/src/main/res/values-v23/themes.xml [hunk 2]
```diff
@@ -66,5 +66,29 @@
 
     <style name="SearchInputViewTheme" parent="StreamUi.SearchInputView"/>
 
+    <style name="AttachmentGalleryStyle" parent="StreamUi.AttachmentGallery">
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/res/values-v23/themes.xml:69]: Let's remove those default styles from sample app

File: stream-chat-android-ui-components-sample/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/AndroidManifest.xml:3]: We can safely remove that

File: stream-chat-android-ui-components/src/main/res/values/styles.xml (hunk 1 above)
Comment (leandroBorgesFerreira) [reply stream-chat-android-ui-components/src/main/res/values/styles.xml:508]: That's a complex one. Discussing with the team...

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**